radixdlt-java/src/main/java/com/radixdlt/client/application/translate/tokens/AtomToTokenTransfersMapper.java

```diff
@@ -1,26 +1,24 @@
 package com.radixdlt.client.application.translate.tokens;
 
+import com.google.gson.JsonParser;
```

Comment:
Would prefer if we didn't unnecessarily change the order of the import statements.  Just messes up diffs in the future.

Reply:
woops sorry, I simply did the auto imports that intelliJ does ctrl + option + O to remove the unused RadixConstants import

I'll let you guys decide on this ordering thing for the lib but in Android I try and do this auto imports shortcut on all files for consistency